autogen/cluster.tf

```diff
@@ -352,6 +352,7 @@ resource "google_container_node_pool" "pools" {
 }
 
 resource "null_resource" "wait_for_cluster" {
+  count = var.skip_provisioners ? 1 : 0
```

Comment:
If you skip provisioners, shouldn't this be count of 0? Please pay attention to quality of work and correct logic.

Reply:
FIxed